example.py

```diff
@@ -7,6 +7,7 @@
 def main():

```

Comment:
We will go through this file again sometime when we fix all of the classes - remind me.

Reply:
You can create a new example with only the new syntax.